scripts/ios/run.sh

```diff
@@ -31,6 +31,13 @@ if [[ ${PUBLISH_PLATFORM} = 'ios' ]]; then
 
     mapbox_time "deploy_ios_stripped"
     ./scripts/ios/publish.sh "${PUBLISH_VERSION}"
+
+    # debug symbols but no Bitcode
+    mapbox_time "package_ios_no_bitcode" \
+    make ipackage-no-bitcode
+
+    mapbox_time "deploy_ios_no_bitcode"
+    ./scripts/ios/publish.sh "${PUBLISH_VERSION}" no-bitcode
```

Comment:
Cool, this all looks good. Appreciate the update here, even though we aren't actively running gate publish step right now on Bitrise.